### DISCUSSION TITLE
Turn off IntelliSense

### DISCUSSION THREAD

**Discussion by Callaghan-Hattingh** [golang/vscode-go#1285] [Help] [answered]
I would just like to know how to turn off/on the IntelliSense for VS code GO. 

I am 100% sure that this is somewhere in the docs but I could not find it, unfortunately.
I would like to turn it off because I find it distracts me.

Thanks for the help.

**Comment by stamblerre** [comment, marked as answer]
Please take a look at https://code.visualstudio.com/docs/editor/intellisense and https://code.visualstudio.com/docs/editor/intellisense#_customizing-intellisense.

**Reply by Callaghan-Hattingh** [reply]
![image](https://user-images.githubusercontent.com/70077927/110532674-ce4f2b00-8125-11eb-8fb9-8ad1cf36a434.png)
 
Thanks for the suggestion but unfortunately I think that I will have to change a setting in gopls settings. 

![gopls settings](https://github.com/golang/vscode-go/blob/master/docs/settings.md#settings-for-gopls)

I am just not sure which setting to change. Please correct me if I am wrong. Thanks

**Reply by stamblerre** [reply]
Intellisense is provided by gopls, but the UI is entirely controlled by VS Code. The second link should show which settings you need to configure.

**Reply by Callaghan-Hattingh** [reply]
Thanks,  changed settings.json in vs code by adding:

    "editor.quickSuggestions": {
        "other": false,
        "comments": false,
        "strings": false
    }
